README.md

```diff
@@ -47,15 +47,17 @@ Legend:
 
 ## Specification Summary
 
-A Functions Framework instantiates web server and invokes function code in response to an HTTP request (`http`) or CloudEvent request (`cloudevent`) depending on the function's signature type. A Functions Framework may optionally support functions with signature type `event` for legacy-style events.
+A Functions Framework instantiates web server and invokes function code in response to an **HTTP** (`http`) or **CloudEvent** (`cloudevent`) request depending on the function's signature type. A Functions Framework may also optionally support functions with signature type `event` for legacy-style events.
 
 The Functions Framework library must be configurable via environment variables and may be configurable via command-line flags:
```

Comment:
[nit] The "may be configurable via command-line flags" part is a little confusing now with the Required YES column in the grid.